src/components/Page.tsx

```diff
@@ -20,29 +20,22 @@ class Page extends Component<PageProps, PageState> {
 
   async componentDidMount() {
     console.log('sdk', this.props.sdk)
-    // Get Space Data
```

Comment:
Can we remove this `componentDidMount`? It looks like the `console.log` is a troubleshooting artifact.